### ISSUE TITLE
Missing Docker Image

### ISSUE THREAD

**Issue by jeffutter** [prometheus/statsd_exporter#30]
This is probably related to #13 .. but the docs currently reference a docker image at prom/statsd-exporter . This project exists on docker but there are no images uploaded. Could we get an image up there?

**Comment by juliusv** [comment]
I found out that prom/statsd-exporter was a "PUBLIC REPOSITORY" on the Docker Hub, while the other Prometheus ones there were "PUBLIC | AUTOMATED BUILD" (what it says at the top). I deleted the public repository one and am now trying to recreate it as an automated build. However, the Docker Hub is being obtuse again and is not showing me an option to select a repo from the Prometheus GitHub org when creating an automated build, but only my personal (juliusv) ones ("Create" in top menu -> "Create Automated Build" -> "Users/Organizations" only shows "juliusv" and not "prometheus"). And this is with my Hub account being linked read/write to my GH account. Docker Hub insights appreciated.
